packages/core/src/components/Checkbox/CheckboxRow.tsx

```diff
@@ -184,5 +195,20 @@ export const SEED_DATA = {
       valuePropName: "status",
       handlerPropName: "onPress",
     }),
+    color: createColorProp({
+      description: "Color for the button (used when the checkbox is checked)",
+    }),
+    uncheckedColor: createColorProp({
+      group: GROUPS.basic,
+      label: "Unselected Color",
+      description: "Color for the button when the checkbox is unchecked",
+      defaultValue: null,
+    }),
+    indeterminateColor: createColorProp({
+      group: GROUPS.basic,
+      label: "Indeterminate Color",
+      description: "Color for the button when the checkbox is indeterminate",
+      defaultValue: null,
```

Comment:
Peter's comment applies to `uncheckedColor` and `indeterminateColor`, too - `defaultValue` and `group` are the same as the defaults applied in `createColorProp`, so they can be removed here.

Reply:
fixed